components/index/Chart.vue

```diff
@@ -27,6 +27,30 @@
         daysArray.push(thatDay.getDate() + '<br/>' + getMonth(thatDay.getMonth()))
       }
       daysArray.reverse()
+
+      const lastDayFinished = new Date(latest).setHours(23, 59, 59, 999) < Date.now()
+      let data = this.txHistory.map(h => h.value)
+      let finishedDaysLength = days
+      if (!lastDayFinished && days > 0) {
+        const unfinishedDayFraction = (latest - new Date(latest).setHours(0, 0, 0, 0)) / (24 * 60 * 60 * 1000)
+        const lastFinishedDayData = data[data.length - 2]
+        const unfinishedDayData = data.pop()
+        const extrapolated = unfinishedDayData + lastFinishedDayData * (1 - unfinishedDayFraction)
+
+        finishedDaysLength = days - 1
+        data.push({
+          x: finishedDaysLength,
+          y: extrapolated,
+          tooltip: `Extrapolated to ~${extrapolated.toFixed(0)} from ${unfinishedDayData}`
+        })
+        data.push(null) // Break line and draw unextrapolated marker too
+        data.push({
+          x: finishedDaysLength,
+          y: unfinishedDayData,
+          marker: { enabled: true }
+        })
```

Comment:
Added extrapolation with tooltip and the real data point below
![0 0 0 0_8081_](https://user-images.githubusercontent.com/3758846/121967007-e14e4880-cd6f-11eb-960e-65917a21c71e.png)